tests/jballerina-debugger-integration-test/src/test/java/org/ballerinalang/debugger/test/adapter/evaluation/ExpressionEvaluationTest.java

```diff
@@ -309,7 +309,7 @@ public void functionCallEvaluationTest() throws BallerinaTestException {
     }
 
     @Override
-    @Test
+    @Test(enabled = false)
```

Comment:
Related issue for disabling this test: https://github.com/ballerina-platform/ballerina-lang/issues/30248

Reply:
@riyafa Ack